pallets/xcm-core-buyer/src/benchmarks.rs

```diff
@@ -45,6 +46,62 @@ mod benchmarks {
         },
     };
 
+    #[benchmark]
+    fn buy_core() {
+        let caller: T::AccountId = whitelisted_caller();
+        assert_ok!(Pallet::<T>::set_relay_xcm_weight_config(
+            RawOrigin::Root.into(),
+            Some(RelayXcmWeightConfigInner {
+                buy_execution_cost: BUY_EXECUTION_COST,
+                weight_at_most: PLACE_ORDER_WEIGHT_AT_MOST,
+                _phantom: PhantomData,
+            }),
+        ));
+
+        let x = 1000u32;
+
+        let para_id = ParaId::from(x + 1);
+        for i in 0..=x {
```

Comment:
I assume you dont want your benchmark to depend on X here, hence why 1000? was this chosen in some arbitrary form?

Reply:
@girazoki Yes. The idea is to have some entries in storage. There is no tangible benefits except maybe slightly more accurate storage read/write timings. 

We can remove it if you want.